cal/cal/static/js/app.js

```diff
@@ -207,7 +284,83 @@ analyticsApp.controller('CategoriesCtrl', function($scope, $http){
       },
     },
   };
+});
 
+function CategoriesDetailCtrl($scope, $http){
+  var categoryUrl = '/v1/colorcategories/' + this.categoryId + '/events';
+  var categoryWeek = '/v1/colorcategories/' + this.categoryId + '/eventWeek';
+  $scope.categoryDetails = [];
+  $scope.categoryEvents = [];
+
+  $http({method: 'GET', url: categoryUrl + '.json' }).
+  success(function successCallback(data) {
+    for (var i = 0; i < data.results.length; i++) {
+      var event = data.results[i]
+      $scope.categoryEvents.push({
+        start: event.start,
+        name: event.name,
+      });
+    }
+  });
+
+  $http({method: 'GET', url: categoryWeek + '.json' }).
+  success(function successCallback(data) {
+    var events = [];
+    for (var i = 0; i < data.length; i++) {
+      var event = data[i];
+      var start = new Date(event[0]);
+      events.push({
+        x: start,
+        y: event[1]
+      });
+    }
+    $scope.categoryDetails.push({
+      values: events,      //values - represents the array of {x,y} data points
+      key: 'Category Graph', //key  - the name of the series.
+      color: '#003057',  //color - optional: choose your own line color.
```

Comment:
Nitpick - comments are unevenly spaced from the end of the text. Either align the start of the comments of keep consistently spacing after the commas